jwtauth/jwtauth.go

```diff
@@ -0,0 +1,56 @@
+// Package jwtauth implements an interface for providing CFSSL
+// authentication.
+package jwtauth
+
+import (
+	//"flag"
+	"fmt"
+	//"io"
+	//"io/ioutil"
+	//"os"
+	"time"
+
+	//"github.com/cloudflare/cfssl/helpers"
+	"github.com/dgrijalva/jwt-go"
+)
+
+/*
+ * Not sure if I need these yet
+ *
+type Provider interface {
+	Token(req []byte) (token []byte, err error)
+	Verify(aReq *AuthenticatedRequest) bool
+}
+
+type AuthenticatedRequest struct {
+	// An Authenticator decides whether to use this field.
+	Timestamp     int64  `json:"timestamp,omitempty"`
+	RemoteAddress []byte `json:"remote_address,omitempty"`
+	Token         []byte `json:"token"`
+	Request       []byte `json:"request"`
+}
+*/
```

Comment:
Looks like Travis is treating this giant block of commented-out code as part of the comment for exported function New() below. Consider remove this block and keep it somewhere for your own reference if you are not sure at the moment.